helix_fhir_client_sdk/responses/fhir_client_protocol.py

```diff
@@ -151,16 +140,6 @@ def page_number(self, page_number: int) -> "FhirClientProtocol": ...
 
     def id_(self, id_: Union[List[str], str] | None) -> "FhirClientProtocol": ...
 
-    async def get_resources_by_id_in_parallel_batches_async(
-        self,
-        concurrent_requests: int,
-        chunks: Generator[List[str], None, None],
-        fn_handle_batch: Optional[HandleBatchFunction],
-        fn_handle_error: Optional[HandleErrorFunction],
-        fn_handle_ids: Optional[HandleBatchFunction] = None,
-        fn_handle_streaming_chunk: Optional[HandleStreamingChunkFunction] = None,
-    ) -> List[Dict[str, Any]]: ...
```

Comment:
why is this method not needed any more?
is there any other alternative method taking care of the action already??

Reply:
we are now using async generators and streaming for this which is easier in python to use.  So these are not needed anymore